packages/foam-vscode/src/features/tags-tree-view/index.ts

```diff
@@ -102,23 +116,27 @@ type TagMetadata = { title: string; uri: URI };
 export class Tag extends vscode.TreeItem {
   constructor(
     public readonly tag: string,
+    public readonly title: string,
     public readonly notes: TagMetadata[]
   ) {
-    super(tag, vscode.TreeItemCollapsibleState.Collapsed);
+    super(title, vscode.TreeItemCollapsibleState.Collapsed);
     this.description = `${this.notes.length} reference${
       this.notes.length !== 1 ? 's' : ''
     }`;
     this.tooltip = getContainsTooltip(this.notes.map(n => n.title));
+    this.tag = tag;
   }
 
   iconPath = new vscode.ThemeIcon('symbol-number');
   contextValue = 'tag';
 }
 
 export class TagSearch extends vscode.TreeItem {
+  public readonly title: string;
   constructor(public readonly tag: string) {
     super(`Search #${tag}`, vscode.TreeItemCollapsibleState.None);
     const searchString = `#${tag}`;
+    this.title = tag.replace(TAG_SEPARATOR, '-');
```

Comment:
out of curiosity: why replace the separator?

Reply:
Turns out it's not necessary anymore due to the other changes. I needed as vscode wouldn't display the item with a forward slash in the title.